wolfssl-3.9.6/README.md

```diff
@@ -0,0 +1,139 @@
+Finds a SEGV and three heap overflow errors in `src/ssl.c`
+in [wolfSSL v.3.9.6](https://www.wolfssl.com/wolfSSL/Home.html).
+
+Crash reproducers included. You may reproduce the crashes by running the individual
+driver on each case:
```

Comment:
It's not! I had forgotten about the `standalone` target. Is there some automatic build rule for this in the harness?